realm/realm-library/src/androidTest/java/io/realm/RealmObjectSchemaTests.java

```diff
@@ -342,6 +342,26 @@ public void testAddPrimaryKeyFieldModifier_illegalFieldTypeThrows() {
         }
     }
 
+    public void testAddPrimaryKeyFieldModifier_duplicateValues() {
```

Comment:
@cmelchior I think we should keep this since there are no tests for `addPrimaryKey()` for duplicate values.

I overlooked?